lib/graphql/fragments.graphql

```diff
@@ -40,6 +42,7 @@ fragment User on  User{
     name
     email
     mobile
+    is_notifiable
```

Comment:
why we are in need to add is_notifiable?

Reply:
to know notification for this user on or off